Wasted/Pages/Client/OrderStatusPage.xaml.cs

```diff
@@ -14,38 +15,24 @@ namespace Wasted
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class OrderStatusPage : ContentPage
     {
-        private DataService Service;
-        public OrderStatusPage(DataService service)
-        {
-            InitializeComponent();
-
-            Service = service;
-
-            foreach (CartDeal cartDeal in service.CartDeals)
-            {
-                service.OrderedDeals.Add(new OrderedDeal(cartDeal, "preparing"));
-            }
-
-            service.CartDeals = new List<CartDeal>();
-            
-            OrderedDealsCollectionView.ItemsSource = service.OrderedDeals;
-        }
-
+        public DataService Service { get; set; }
         public OrderStatusPage()
         {
-            InitializeComponent();
-            
             Service = DependencyService.Get<DataService>();
-            
-            OrderedDealsCollectionView.ItemsSource = Service.OrderedDeals;
         }
         
         /// <summary>
         /// Sets OrderedDealsCollectionView with OrderedDeals
         /// </summary>
         protected override void OnAppearing()
         {
+            InitializeComponent();
```

Comment:
Does not work, as opening the order status page the second time crashes the app.